develop/dev-guide-proxysql-integration.md

```diff
@@ -1,682 +1,1116 @@
 ---
-title: Integrate TiDB with ProxySQL
-summary: Introduce how to integrate TiDB with ProxySQL step by step.
+title: ProxySQL Integration Guide
+summary:  Learn how to integrate TiDB Cloud and TiDB (self-hosted)  with ProxySQL
 ---
 
 # Integrate TiDB with ProxySQL
 
-This document describes how to integrate **TiDB** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. For more information, refer to:
+This document provides a high-level introduction to ProxySQL, describes how to [integrate ProxySQL with TiDB](#option-1---integrate-tidb-serverless-with-proxysql), and demonstrates the key integration benefits through the [scenario of query routing](#typical-scenario).
```

Comment:
```suggestion
This document provides a high-level introduction to ProxySQL, describes how to [integrate ProxySQL with TiDB](#option-1----integrate-tidb-serverless-with-proxysql), and demonstrates the key integration benefits through the [scenario of query routing](#typical-scenario).
```

Reply:
Fixed. Change the weird title space instead.